addons/sourcemod/scripting/vsh/abilities/ability_rage_hop.sp

```diff
@@ -118,15 +121,15 @@ methodmap CRageHop < SaxtonHaleBase
 			Format(sSound, sizeof(sSound), "weapons/airstrike_small_explosion_0%i.wav", GetRandomInt(1,3));
 			
 			if(this.bSuperRage)
-				TF2_Explode(this.iClient, vecExplosionOrigin, this.flBombDamage * 1.5, this.flBombRadius, "heavy_ring_of_fire", sSound);
+				TF2_Explode(this.iClient, vecExplosionOrigin, this.flBombDamage * 1.5, this.flBombRadius * 1.25, "heavy_ring_of_fire", sSound);
 			else
 				TF2_Explode(this.iClient, vecExplosionOrigin, this.flBombDamage, this.flBombRadius, "heavy_ring_of_fire", sSound);
 			
 			for (int i = 1; i <= MaxClients; i++)
 			{
 				if (IsClientInGame(i) && IsPlayerAlive(i) && GetClientTeam(i) > 1 && GetClientTeam(i) != iTeam)
 				{
-					if (IsClientInRange(i, vecExplosionOrigin, 200.0))
+					if (IsClientInRange(i, vecExplosionOrigin, this.flBombRadius) || (IsClientInRange(i, vecExplosionOrigin, this.flBombRadius * 1.25) && this.bSuperRage))
```

Comment:
I would be more happy if radius is calculated to local variable and test 1 `IsClientInRange`, rather than having 2 `IsClientInRange`